tpm2/tpm2.go

```diff
@@ -4,8 +4,10 @@ package tpm2
 import (
 	"bytes"
 	"encoding/binary"
+	"io"
```

Comment:
I'm not sure this import is needed - does `goimports` add it?

Reply:
This was due to an erroneous pull and merge, I believe it is fixed now?

Reply:
Perfect, thank you!